articles/container-registry/container-registry-oras-artifacts.md

```diff
@@ -31,7 +31,7 @@ ORAS Artifacts support is not available in the government or China clouds, but a
 
 ## ORAS installation
 
-Download and install a preview ORAS release for your operating system. See [ORAS installation instructions][oras-install-docs] for how to extract and install the file for your operating system. This article uses ORAS CLI 0.14.1 to demonstrate how to manage supply chain artifacts in ACR.
+Download and install a preview ORAS release for your operating system. See [ORAS installation instructions][oras-install-docs] for how to extract and install the file for your operating system. This article uses ORAS CLI 0.16.0 to demonstrate how to manage supply chain artifacts in ACR.
```

Comment:
We need to rename ORAS Artifacts to OCI artifacts. Also this doc can go out only after the release is complete. 

@northtyphoon - Do we still need zone redundancy going forward?  

@FeynmanZhou If not we need to remove LIne 20

Reply:
@sajayantony  It is not required now after I confirmed it with @northtyphoon. So I removed Line 20.